packages/client/src/rpc/modules/debug.ts

```diff
@@ -457,4 +460,25 @@ export class Debug {
     this.client.config.logger.configure({ level: logLevels[level] })
     return `level: ${this.client.config.logger.level}`
   }
+
+  /**
+   * Returns an RLP-encoded block
+   * @param blockOpt Block number or tag
+   */
+  async setHead(params: [string]) {
+    const [blockOpt] = params
+    if (blockOpt === 'pending') {
+      throw {
+        code: INVALID_PARAMS,
+        message: `"pending" is not yet supported`,
```

Comment:
```suggestion
        message: `"pending" is not supported`,
```

We should never support pending for this operation.  It doesn't make sense to set our head to a "pending" block that hasn't been included in the chain.